content/blog/2020/08/2020-08-29-git-performance-improvement-phase3.adoc

```diff
@@ -0,0 +1,126 @@
+---
+layout: post
+title: "Git Plugin Performance Improvement: Final Phase and Release"
+tags:
+- plugins
+- git
+- platform-sig
+- developer
+- performance
+- gsoc
+- gsoc2020
+author: rishabhbudhouliya
+opengraph:
+  image: /images/post-images/2020/07-git-plugin-performance-improvement/gsoc-git-opengraph.png
+---
+
+Since the beginning of the project, the core value which drove its progress was "To enhance the user experience for running Jenkins jobs by reducing the overall execution time".
+
+To achieve this goal, we laid out a path: 
+
+* Compare the two existing git implementations i.e CliGitAPIImpl and JGitAPIImpl using performance benchmarking
+* Use the results to create a feature which would improve the overall performance of git plugin
+* Also, fix existing user reported performance issues
+
+Let's take a journey to understand how we've build the new features. If you'd like to skip the journey part, you can directly skip to the end section to see what we've done!
+
+== Journey to the release of Git Plugin 4.4.0 and Git Client Plugin 3.4.0
```

Comment:
The long title wraps on the site when formatted on my wide monitor.  You might consider reducing the length of the title.
```suggestion
== Journey to release
```